frontend/src/components/VFeatureNotice/VFeatureNotice.vue

```diff
@@ -0,0 +1,57 @@
+<script setup lang="ts">
+import VLink from "~/components/VLink.vue"
+
+defineProps<{
+  seeMoreHref: string
+}>()
+
+const emit = defineEmits<{
+  click: [MouseEvent]
+}>()
+
+const handleClick = (e: MouseEvent) => {
+  emit("click", e)
+}
+</script>
+
+<template>
+  <div
+    class="border-gradient flex h-10 w-[23.125rem] flex-row items-center gap-2 rounded-full border border-double border-tx bg-origin-border pe-4 ps-2"
```

Comment:
```w-[23.125rem]
```

This will need a media query and some adjustment for smaller mobile viewports:

![image](https://github.com/user-attachments/assets/83d35f45-eaee-4662-b306-750605601d70)

Reply:
I added the change to make this take up the full width on mobile phones (with x-margins) which aligns it with the hero text on both sides. Now it also wraps text inside without overflowing.